vm/systemSmartContracts/auction.go

```diff
@@ -733,6 +735,7 @@ func (s *stakingAuctionSC) saveRegistrationData(key []byte, auction *AuctionData
 func (s *stakingAuctionSC) getStakedData(key []byte) (*StakedData, error) {
 	vmOutput, err := s.executeOnStakingSC([]byte("get@" + hex.EncodeToString(key)))
 	if err != nil {
+		s.eei.AddReturnMessage("cannot executed get: error " + err.Error())
```

Comment:
"cannot execute get: " + err.Error() is enough

Reply:
done.